ledger-service/http-json/src/main/resources/logback.xml

```diff
@@ -20,7 +20,7 @@
     <logger name="io.grpc.netty" level="WARN" />
     <logger name="ch.qos.logback" level="WARN" />
 
-    <root level="INFO">
+    <root level="${ROOT_LOG_LEVEL_JSON_API:-INFO}">
```

Comment:
I don't think we should make this change here. How config is done is kind of a separate concern. You could put note in the PR description about how to modify this if you wanted to assist a reviewer in playing around with it.

Reply:
I definitely agree is separate concern, but as well I though it wasn't harm or even noticed that this change was in place, but instead give more flexibility for future issue resolutions... jmho, but leaving it as it was 👍🏾